src/components/forms/DocumentUploadForm/UploadFile/UploadFileBack/FileBackSelected.vue

```diff
@@ -1,15 +1,12 @@
 <template>
-  <div>
-    <upload-title :title="$t('components.uploadDocument.backIsSelected')" />
-    <file-selected
-      :file="file"
-      v-on="$listeners"
-    />
-  </div>
+  <file-selected
```

Comment:
Maybe rename component to `selected-file`?

Reply:
not sure, 
in folder when we have different components, and when you name `file` in the beginning of component - will be grouped different components in list.
for example `shared` folder have:
```
FileSelected
FileName
UploadTitle
```
with rename it will be like:
```
FileName
UploadTitle
SelectedFile
```

Reply:
We can use subfolders. File \ SelectedFile, File \ UploadTitle, File \ Filename. Just in case.

Reply:
We can use subfolders. File \ SelectedFile, File \ UploadTitle, File \ Filename. Just in case.

Reply:
I think folder structure less valuable than places where component will be used